src/scalper.ts

```diff
@@ -860,30 +907,71 @@ export class Scalper {
     const backAskID = new BN(orderIDBase * 2 + 1);
     const gammaIds = [bidID.toString(), askID.toString(), backBidID.toString(), backAskID.toString()];
 
-    if (this.serumVialClient.checkSerumVial() != WebSocket.OPEN) {
+    if (this.serumVialClient.checkSerumVial() !== WebSocket.OPEN) {
       this.serumVialClient.openSerumVial();
     }
 
     let gammaFillQty = 0;
+    let amountGamma = 0;
```

Comment:
This doesn't change and a fill can't be found before the orders are place. It's just safer like this?

Reply:
correct. you shouldnt use a variable before you define it. It is used in the callback.
I will check if the callback can be moved later when I get to actually cleaning up logic in the risk manager